turbinia/workers/grep.py

```diff
@@ -36,11 +38,14 @@ def run(self, evidence, result):
     Returns:
         TurbiniaTaskResult object.
     """
-
-    patterns = evidence.config.get('filter_patterns')
-    if not patterns:
+    patterns_file = self.recipe['filter_patterns_file']
```

Comment:
I can't seem to find a way to respond to your earlier response, so I'm quoting it here:

> References to patterns file eliminated from turbiniactl and recipe parser.

> Made some preliminary modifications to the grep task to allow it to work with the task_config as well as recipes, moved the logic to read a patterns file to this task.

Sorry for the confusion here, but I think we'll still have to read/parse the file on the client side and then pass in the entire list into the recipe as just `filter_patterns` rather than look for a file from the worker.  This is because the worker may not have access to the file at the same path.

> Thinking about how this would work with docker, should we give "TextFile" and "PlasoCsvFile" source_path attributes to make the file itself accesible to the task?

I think if we parse the file client side and then pass in the list to the recipe as mentioned above that this will solve this case as well.